### ISSUE TITLE
xstyle.js is empty (0 bytes)

### ISSUE THREAD

**Issue by speich** [kriszyp/xstyle#66]
The file https://github.com/kriszyp/xstyle/blob/master/xstyle.js is empty. Did building fail?

**Comment by kriszyp** [comment]
I don't think that is a built file, I think the functionality was moved elsewhere.
